File: Python3/Electrical Outlets.py
#https://open.kattis.com/problems/electricaloutlets
def calc(arr):
    sum = 0
    for i in range(int(arr[0])):
        sum += int(arr[i+1])
    return sum - int(arr[0]) + 1
n = int(input())
nums = []
for i in range(n):
    nums += [input().split(" ")]
for thing in nums:
    print(calc(thing))

File: C/Electrical Outlets.c
//https://open.kattis.com/problems/electricaloutlets
//MASSIVE THANKS TO https://github.com/minidomo/Kattis/blob/master/C/electricaloutlets.c
#include <stdio.h>
//#include <string.h>
//#include <stdlib.h>

int main() {
	int n;
	scanf("%d", &n);
	while(n--)
	{
		int num;
		scanf("%d", &num);
		int ans;
		ans = 1 - num;
		while (num--)
		{
			int c;
			scanf("%d", &c);
			ans += c;
		}
		printf("%d\n", ans);
	}
  return 0;
}
